src/run_all.py

```diff
@@ -25,8 +28,8 @@ def main(selected_polygon_gdf: gpd.GeoDataFrame, modules_with_log_levels: Dict[M
     # Define a dictionary mapping each module to its log level
     module_to_log_level = {
         run: LogLevel.DEBUG,
-        lidar_metadata_in_db: LogLevel.DEBUG,
-        dem_metadata_in_db: LogLevel.DEBUG,
+        datasets: LogLevel.DEBUG,  # only need to run it one time to initiate db.dataset table
```

Comment:
It is giving this error:

Traceback (most recent call last):
  File "C:\Users\sli229\Projects\Digital-Twins\src\run_all.py", line 40, in <module>
    main(sample_polygon, module_to_log_level)
  File "C:\Users\sli229\Projects\Digital-Twins\src\run_all.py", line 24, in main
    module.main(selected_polygon_gdf, log_level=log_level)
  File "C:\Users\sli229\Projects\Digital-Twins\../NewZeaLiDAR\newzealidar\datasets.py", line 336, in main
    utils.map_dataset_name(engine, instructions_file)
  File "C:\Users\sli229\Projects\Digital-Twins\../NewZeaLiDAR\newzealidar\utils.py", line 231, in map_dataset_name
    with open(instructions_file, 'r') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'configs\\instructions.json'

Reply:
`.env` needs to be modified following the `.env.template`.